extensions/gcp/processors/DeleteGCSObject.h

```diff
@@ -0,0 +1,65 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <string>
+#include <memory>
+
+#include "GCSProcessor.h"
+#include "core/logging/Logger.h"
+#include "core/logging/LoggerConfiguration.h"
+#include "../controllerservices/GCPCredentialsControllerService.h"
+#include "google/cloud/storage/client.h"
+#include "google/cloud/storage/retry_policy.h"
+#include "utils/Enum.h"
+
+namespace org::apache::nifi::minifi::extensions::gcp {
+
+class DeleteGCSObject : public GCSProcessor {
+ public:
+  explicit DeleteGCSObject(const std::string& name, const utils::Identifier& uuid = {})
+      : GCSProcessor(name, uuid, core::logging::LoggerFactory<DeleteGCSObject>::getLogger()) {
+  }
+  DeleteGCSObject(const DeleteGCSObject&) = delete;
+  DeleteGCSObject(DeleteGCSObject&&) = delete;
+  DeleteGCSObject& operator=(const DeleteGCSObject&) = delete;
+  DeleteGCSObject& operator=(DeleteGCSObject&&) = delete;
+  ~DeleteGCSObject() override = default;
```

Comment:
the base class `Connectable` already has these methods deleted

Reply:
:+1: removed them in https://github.com/apache/nifi-minifi-cpp/pull/1297/commits/e09d82437a699e9484e38c5e20cf1ee50c06eb59